components/user_settings/import_theme_modal.tsx

```diff
@@ -64,15 +64,36 @@ class ImportThemeModal extends React.PureComponent<WrappedComponentProps, State>
             return;
         }
 
+        /*
+         * index mapping of slack theme format (index => slack-property name)
+         *
+         * |-------|-------------------------|-------------------------|
+         * | index | Slack theme-property    | MM theme-property       |
+         * |-------|-------------------------|-------------------------|
+         * |   0   | Column BG               | sidebarBg               |
+         * |   1   | ???                     | sidebarHeaderBg         |
```

Comment:
I assume we will have a value for this?

Reply:
Yes, we do, but there is no value name given in Slack

Reply:
<img width="556" alt="Screenshot 2021-02-11 at 14 37 18" src="https://user-images.githubusercontent.com/32863416/107643671-c01d0480-6c76-11eb-913b-23564d144bd6.png">

The value is there, but there is no corresponding field for it above.

Reply:
Understand, maybe we add something like Sidebar Header or Column header just for our own clarity. 0/5 on this

Reply:
well, we could, but since we are mapping it against our own `sidebarHeaderBg` property it feels obsolete.

How about we just remove the `???` and just leave it blank (0/5) or add `---` (1/5)?